operations/iuf/stages/deliver_product.md

```diff
@@ -40,6 +40,20 @@ Artifacts will be uploaded to the following locations based on entries specified
 | VCS Content                   | `vcs`                             |
 | IMS Images and Recipes        | `ims`                             |
 
+The following table describes upload behavior when the artifact being uploaded already exists on the system.
```

Comment:
This table describes upload behavior of artifacts already present on the system.

Reply:
Fixed in next update.